Automattic/jetpack
# Jetpack Setup Wizard: Add Recommended Features Page

#### Changes proposed in this Pull Request:

This PR builds on the work in #15579 by adding recommended features page.

Figma design:
<img width="853" alt="image" src="https://user-images.githubusercontent.com/42627630/81992928-3b7b5800-960a-11ea-90b1-f0c1c7d9080a.png">

Figma design mobile:
<img width="180" alt="image" src="https://user-images.githubusercontent.com/42627630/81992956-4df59180-960a-11ea-8403-2f087dd94d67.png">

Current state:
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/42627630/81993130-afb5fb80-960a-11ea-8bcd-adfba35ff122.png">

Current design mobile:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/42627630/81993279-fe639580-960a-11ea-8d07-0ddadfab6500.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Project: p1HpG7-8BF-p2
Design: pbtFPC-ol-p2
Toggle states: https://cloudup.com/c03Bs_MSJSk

#### Testing instructions:





1. Add the following to the end of wp-config.php: add_filter( 'jetpack_show_setup_wizard', '__return_true' );
2. Visit `/wp-admin/admin.php?page=jetpack#/setup/features`.

For all the features, or at least as many as you can stand:
1. Test the toggle to ensure the module turns on and off if applicable, or the toggle is disabled otherwise.
2. For paid features verify that free plans see upgrade buttons and that toggling "on" takes you to upgrade.
3. Test "View options" links and "Configure" buttons.
4. Pay particular attention to paid features and upgrade paths.
5. Test responsive behavior of the page.

#### Proposed changelog entry for your changes:
* None needed; the entire setup wizard project should get one entry.
